db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161118012333) do
+ActiveRecord::Schema.define(version: 20161115154942) do
```

Comment:
Yikes! Should I change it back to the line in red? I ran rake db:migrate in a desperate problem-solving attempt for something unrelated... probably a mistake.